tools/packaging/kata-deploy/scripts/kata-deploy.sh

```diff
@@ -168,10 +170,10 @@ function get_kata_containers_runtime_path() {
 	local runtime_path
 	case "$shim" in
 		cloud-hypervisor | dragonball | qemu-runtime-rs)
-			runtime_path="/opt/kata/runtime-rs/bin/containerd-shim-kata-v2"
+			runtime_path="$dest_dir/runtime-rs/bin/containerd-shim-kata-v2"
 			;;
 		*)
-			runtime_path="/opt/kata/bin/containerd-shim-kata-v2"
+			runtime_path="$dest_dir/bin/containerd-shim-kata-v2"
```

Comment:
nit: For consistency (and because I've been burnt by parameter expansion in the past), could we make these `${dest_dir}`?